config/registry/default_schema.yaml

```diff
@@ -0,0 +1,719 @@
+          sensor; that requires specifying an exposure or visit as well).
+        link:
+        - 
+          name: sensor
+          type: string
+          foreign_key: Sensor.sensor
+        tables:
+          Sensor:
+            doc: >
+              Metadata and grouping information, allowing
+              sensors to be identified by something other than their numbers in user
+              expressions of what to process.
+            columns:
+            - 
+              name: camera
+              type: string
+              primary_key: true
+              nullable: false
+            - 
+              name: sensor
+              type: int
+              primary_key: true
+              nullable: false
+            - 
+              name: name
+              type: string
+              primary_key: true
+            - 
+              name: group
+              type: string
+            - 
+              name: purpose
+              type: string
+            foreignKeys:
+            - 
+              src: camera
+              tgt: Camera.camera
+    
+      Visit:
+        dependencies:
+          required:
+          - Camera
+          optional:
+          - PhysicalFilter
+        doc: >
+          A sequence of observations processed together, comprised of one or more
+          Exposures.
+        link:
+        - 
+          name: visit
+          type: int
+          foreign_key: Visit.visit
+        tables:
+          Visit:
+            doc: >
+              Visit metadata, allowing them to be identified by something other than their numbers.
+            columns:
+            - 
+              name: camera
+              type: string
+              primary_key: true
+              nullable: false
+            - 
+              name: visit
+              type: int
+              primary_key: true
+              nullable: false
+            - 
+              name: physical_filter
+              type: string
+              nullable: false
+            - 
+              name: datetime_begin
+              type: datetime
+              doc: >
+                This should be the same as the datetime_begin field of the first Exposure
+                associated with this Visit.
+            - 
+              name: exposure_time
+              type: float
+              doc: >
+                This should cover the full time from the start of the first Exposure to
+                the end of the last one.
+            - 
+              name: region
+              type: blob
+            foreignKeys:
+            - 
+              src: camera
+              tgt: Camera.camera
+            - 
+              src:
+              - camera
+              - physical_filter
+              tgt:
+              - PhysicalFilter.camera
+              - PhysicalFilter.physical_filter
+    
+      Exposure:
+        dependencies:
+          required:
+          - Camera
```

Comment:
Also:
```
optional:
- Visit
```
right?

(Just making sure this an accidental omission, not something about the structure I don't understand.)

Reply:
Yes, this was an accidental omission (there may be more).